### DISCUSSION TITLE
Opening Local Workspace from the Web

### DISCUSSION THREAD

**Discussion by DamianRivas** [eclipse-theia/theia#9836] [Q&A]
### Description

I have a question about deploying our own Theia IDE. In the desktop environment it's fairly obvious how workspace management will work, but how is workspace management expected to work in the cloud? We are aware of Gitpod, but what other alternatives are there to running Theia in the cloud?

**Comment by akosyakov** [comment]
> but how is workspace management expected to work in the cloud?

Generally, it is out of scope of Theia. Theia is a framework to implement IDEs. How it is deployed and used depends on environments.

> We are aware of Gitpod, but what other alternatives are there to running Theia in the cloud?

As far as I know Gitpod is implemented based on Kubernetes infra of the Google Cloud Platform.
You are better to ask on the Gitpod community for details: https://spectrum.chat/gitpod

Soon, there is also going to be Gitpod Enterprise supporting self-hosting on the vanilla Kubernetes: https://www.gitpod.io/pricing/#enterprise
